pkgs/by-name/cr/crystal-dock/package.nix

```
@@ -0,0 +1,42 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  kdePackages,
+  qt6,
+}:
+stdenv.mkDerivation rec {
+  pname = "crystal-dock";
+  version = "2.2";
+
+  src = fetchFromGitHub {
+    owner = "dangvd";
+    repo = "crystal-dock";
+    rev = "v${version}";
+    sha256 = "sha256-c5Kae2cZ/DoJ972VT4kQWNUr2cF6Noy3nPIChWok/BA=";
```

Using `hash` rather than `sha256` provides more flexibility to change the hashing algorithm in the future:
```suggestion
    hash = "sha256-c5Kae2cZ/DoJ972VT4kQWNUr2cF6Noy3nPIChWok/BA=";
```